db/schema.rb

```diff
@@ -31,13 +31,13 @@
     t.integer "selection_type", default: 0, null: false
     t.integer "min_players_per_team"
     t.integer "max_players_per_team"
+    t.text "image_data"
```

Comment:
Please revert the changes you made in this file as they're only adding noise.

Reply:
Whoah where did that came from 😮 - Will fix

Reply:
Happens sometimes when running migrations.

Reply:
Done